src/docfx/docset/Docset.cs

```diff
@@ -312,7 +312,6 @@ private static async Task<(List<Error>, Dictionary<string, Docset>)> LoadDepende
                 var (loadErrors, subConfig) = ConfigLoader.TryLoad(dir, options, locale);
                 errors.AddRange(loadErrors);
 
-                subLock = subLock ?? await Docs.Build.DependencyLock.Load(dir, subConfig.DependencyLock);
                 result.TryAdd(PathUtility.NormalizeFolder(name), await Create(report, dir, locale, subConfig, options, subLock, isDependency: true));
```

Comment:
This method could then be sync

Reply:
there is still `await Create` in the next line